pages/badge/[id].js

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { Avatar, Card, Col, List, Row, Skeleton, Tag, Typography } from "antd";
+import { withAuth } from "~/components/Auth";
+import AppLayout from "~/components/layouts/AppLayout";
+import * as api from "~/lib/utils/api";
+
+import styles from "./style.module.css";
+
+const colors = {
```

Comment:
Any other good idea? @miguelbrandao

Reply:
Yeah, that seems like a good solution for now. maybe we can improve it when/if we implement internationalization. But why not store it in the database in portuguese?

Also, this will be usefull wverywhere where the belt is shown, so maybe it should be somewhere in the `utils` folder?

Reply:
Yes, I guess for now it should be in the utils :muscle: